### DISCUSSION TITLE
Add swipe menu to the watch face's right

### DISCUSSION THREAD

**Discussion by realpixelcode** [InfiniTimeOrg/InfiniTime#1596] [Ideas]
I don't like that important features of the PineTime are "hidden" in a cluttered 2-page list of tiny, tiny icons. 

Instead, it would be great if one could access important apps like timer, alarm and step counter by swiping from the watch face to the right.

Each of those apps would have its own page. Or there could be a quick menu with 4 icons like on the left with brightness, flashlight, vibrations and settings.

That way, the user wouldn't have to spend unnecessary time looking for the most used features. Other stuff like the games could remain in the list of course.

**Comment by kieranc** [comment]
PR #708 seems to fit the bill
